release/staging-build.yml

```diff
@@ -83,6 +83,12 @@ batch:
       env:
         variables:
           PROJECT_PATH: projects/aws/image-builder
+    - identifier: aws_observability_aws_otel_collector
```

Comment:
I think we can skip adding this to the staging build. There is a key you can add to the makefile so it will be ignored. Check the emissary makefile

Reply:
Done